src/circuit/auto-relay.js

```diff
@@ -224,7 +233,32 @@ class AutoRelay {
       }
     }
 
-    // TODO: Try to find relays to hop on the network
+    // Try to find relays to hop on the network
+    try {
+      const cid = await namespaceToCid(RELAY_RENDEZVOUS_NS)
+      for await (const provider of this._libp2p.contentRouting.findProviders(cid)) {
```

Comment:
We don't have support to do this right now, but it would be great to actually prioritize the results of this based on latency of the connection. Worth adding an issue for a future enhancement here I think

Reply:
Yeah, that would be good! I can create an issue for that. We should get that after the `libp2p.discovery` API, also with the rendezvous

Reply:
https://github.com/libp2p/js-libp2p/issues/758